apps/mull-ui/src/app/pages/home/discover/discover.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { IEvent } from '@mull/types';
+import { EventCard } from '../../../components';
+import { gql, useQuery } from '@apollo/client';
+
+import '../home-discover.scss';
+
+interface DiscoverData {
+  discoverEvents: Partial<IEvent>[];
+}
+
+export const GET_DISCOVER_EVENTS = gql`
+  query DiscoverEvents($discoverEventsUserId: Int!) {
+    discoverEvents(userId: $discoverEventsUserId) {
+      id
+      title
+      description
+      startDate
+      endDate
+    }
+  }
+`;
+
+export const DiscoverPage = () => {
```

Comment:
`discover.(tsx|spec.tsx|scss)` files should be renamed to better fit the `DiscoverPage` name. Something like `discover-page`